apps/src/lib/client/tx.rs

```diff
@@ -1140,28 +1142,12 @@ pub async fn broadcast_tx(
         _ => panic!("Cannot broadcast a dry-run transaction"),
     };
 
-    let websocket_timeout =
-        if let Ok(val) = env::var(ENV_VAR_ANOMA_TENDERMINT_WEBSOCKET_TIMEOUT) {
-            if let Ok(timeout) = val.parse::<u64>() {
-                Duration::new(timeout, 0)
-            } else {
-                Duration::new(300, 0)
-            }
-        } else {
-            Duration::new(300, 0)
-        };
-
-    let mut wrapper_tx_subscription = TendermintWebsocketClient::open(
-        WebSocketAddress::try_from(address.clone())?,
-        Some(websocket_timeout),
-    )?;
+    let rpc_cli = HttpClient::new(address)?;
 
-    let response = wrapper_tx_subscription
-        .broadcast_tx_sync(tx.to_bytes().into())
-        .await
-        .map_err(|err| WsError::Response(format!("{:?}", err)))?;
+    // TODO: timeout?
+    let response = rpc_cli.broadcast_tx_sync(tx.to_bytes().into()).await?;
 
-    wrapper_tx_subscription.close();
+    drop(rpc_cli);
```

Comment:
why is this needed? It will drop in 10 lines anyway

Reply:
If you really need it to drop, use an anonymous scope and return the `response` var out of it

Reply:
no reason, I was experimenting with this to fix websockets errors. this is a remnant of that code.